verifier/verifier.go

```diff
@@ -838,6 +838,10 @@ func revocationFinalResult(certResults []*revocationresult.CertRevocationResult,
 				revokedCertSubject = problematicCertSubject
 			}
 		}
+
+		if i < len(certResults)-1 && certResult.Result == revocationresult.ResultNonRevokable {
+			logger.Warnf("Certificate #%d in the chain with subject %v neither has an OCSP nor a CRL revocation method.", (i + 1), cert.Subject.String())
```

Comment:
We should only display the warning under `strict` mode?

Reply:
In other words, I don't see a particular reason to set it as a warning under `log` mode.

Reply:
Alternative methods exist for verifying revocation status, such as through the use of plugins. Therefore, displaying this warning may be inaccurate.

Reply:
https://github.com/notaryproject/notation-go/blob/5a323330d0392dcf703930c1ff4b46a6197ccdc0/verifier/verifier.go#L505-L518

If the plugin has `CapabilityRevocationCheckVerifier`, it will call the plugin instead of triggering this code.